crates/typst-layout/src/inline/linebreak.rs

```diff
@@ -543,7 +543,14 @@ fn raw_ratio(
 ) -> f64 {
     // Determine how much the line's spaces would need to be stretched
     // to make it the desired width.
-    let delta = available_width - line_width;
+    let delta = {
+        let delta = available_width - line_width;
+        if delta.abs().to_pt() < 0.00001 {
```

Comment:
`delta.approx_eq(Abs::zero())` would be a little cleaner

Reply:
Changed.